website/docs/ecosystem/agentops.md

```diff
@@ -1,32 +1,80 @@
+
+At a high level, AgentOps gives you the ability to monitor LLM calls, costs, latency, agent failures, multi-agent interactions, tool usage, session-wide statistics, and more. For more info, check out the [AgentOps Repo](https://github.com/AgentOps-AI/agentops).
+
+> Create an account and API key at [AgentOps.ai](https://agentops.ai/)
+
+<details open>
+  <summary>Agent Dashboard</summary>
+   <img src="https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/overview.png" style="width: 90%;" alt="Agent Dashboard"/>
+</details>
+
+<details>
+  <summary>Session Statistics</summary>
+   <img src="https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/session.png" style="width: 90%;" alt="Agent Dashboard"/>
+</details>
+
+<details>
+  <summary>Session Replay</summary>
+   <img src="https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/replay.png" style="width: 90%;" alt="Agent Dashboard"/>
+</details>
+
+
+## Installation
 
-> Create an account at [AgentOps.ai](https://agentops.ai/)
 AgentOps works seamlessly with applications built using Autogen.
 
-Install AgentOps with
+1. **Install Autogen with AgentOps**
 ```bash
 pip install pyautogen[agentops]
 ```
 
+2. **Create an API Key:**
+Create a user API key here: [Create API Key](https://app.agentops.ai/account)
+
+3. **Configure Your Environment:**
+Add your API key to your environment variables
+
+```
+AGENTOPS_API_KEY=<YOUR_AGENTOPS_API_KEY>
+```
+
+4. **Initialize AgentOps**
+
 To start tracking all available data on Autogen runs, simply add two lines of code before implementing Autogen.
 
 ```python
 import agentops
-agentops.init() # AgentOps API key named AGENTOPS_API_KEY in environment
-# or
-agentops.init(api_key='you-api-key')
+agentops.init()
+# or agentops.init(api_key="your-api-key-here")
 ```
 
-After initializing AgentOps, Autogen will now start automatically tracking
-- LLM calls
-- Agent names and actions
-- Tool usage
-- Correspondence between agents
-- Errors
-- Token usage and costs
-
-### Example Implementation
-[AgentOps-Notebook](/docs/notebooks/agentchat_agentops)
-[AgentOps Documentation](https://docs.agentops.ai/v1/quickstart)
\ No newline at end of file
+After initializing AgentOps, Autogen will now start automatically tracking your agent runs.
+
+## Features
+
+- **LLM Costs**: Track spend with foundation model providers
+- **Replay Analytics**: Watch step-by-step agent execution graphs
+- **Recursive Thought Detection**: Identify when agents fall into infinite loops
+- **Custom Reporting:** Create custom analytics on agent performance
+- **Analytics Dashboard:** Monitor high level statistics about agents in development and production
+- **Public Model Testing**: Test your agents against benchmarks and leaderboards
+- **Custom Tests:** Run your agents against domain specific tests
+- **Time Travel Debugging**: Restart your sessions from checkpoints
```

Comment:
"Cache your sessions and rerun live calls from checkpoints"

Reply:
"Restart your sessions" isn't clear

Reply:
How about:
>- **Time Travel Debugging**:  Save snapshots of session states to rewind and replay agent runs from chosen checkpoints.